### PR TITLE
Add blog article for Langchain

### PR DESCRIPTION
**Pull request by tobi-brenner**

### CODE REVIEW

File: docs/presentations/articles/langchain.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Langchain: Simplifying LLM-Powered Application Development
+Author: Tobias Brenner
+
+## TL;DR¶
```
Comment (pkeilbach) [inline docs/presentations/articles/langchain.md:4]: The `¶` character seems weird. Can you remove it throughout the document?

image

**Review by pkeilbach — CHANGES_REQUESTED** [review]
Looks good so far, one comment regarding the `¶` character ✌🏼

File: docs/presentations/articles/langchain.md (hunk 1 above)
Comment (tobi-brenner) [reply docs/presentations/articles/langchain.md]: @pkeilbach  The ¶ signs are removed now 😄

**Review by pkeilbach — APPROVED** [review]
LGTM! 🚀

**Pull request merged**